apps/explorer/src/app/routes/network-parameters/network-parameters.test.tsx

```diff
@@ -0,0 +1,39 @@
+import { render, screen } from '@testing-library/react';
+import { NetworkParametersTable } from './network-parameters';
+import type { NetworkParametersQuery } from './__generated__/NetworkParametersQuery';
+
+describe('NetworkParametersTable', () => {
+  it('renders correctly when it has network params', () => {
+    const data: NetworkParametersQuery = {
+      networkParameters: [
+        {
+          __typename: 'NetworkParameter',
+          key: 'market.liquidityProvision.minLpStakeQuantumMultiple',
+          value: '1',
+        },
+        {
+          __typename: 'NetworkParameter',
+          key: 'market.fee.factors.infrastructureFee',
+          value: '0.0005',
+        },
+      ],
+    };
+    const { container } = render(<NetworkParametersTable data={data} />);
+    expect(screen.getByTestId('network-param-header')).toHaveTextContent(
+      'Network Parameters'
+    );
+    const rows = container.getElementsByTagName('dl');
+
+    expect(rows[0].firstChild).toHaveTextContent(
```

Comment:
chore (blocking): I think these two rows are testing the same thing?

Did you mean to assert the value of the `<dd>` element?